platforms/xilinx_alveo_u250/cl_firesim/design/bitstream_config.xdc

```diff
@@ -0,0 +1,12 @@
+# The majority of these constraints are redudant as them come with the u250 given XDC
```

Comment:
Technically this file will also be used with the U280. Are the values the same between the two?

Reply:
Yes, they are the same. write_cfg_mem parameters are also generalized across Alveo